internal/resource/sysfs-device.go

```diff
@@ -64,3 +64,8 @@ func (d vfioDevice) IsMigEnabled() (bool, error) {
 func (d vfioDevice) IsMigCapable() (bool, error) {
 	return false, nil
 }
+
+func (d vfioDevice) GetClass() (string, error) {
+	class := fmt.Sprintf("0x0%x", d.nvidiaPCIDevice.Class)
```

Comment:
Does  the following make more sense here:
```suggestion
	class := fmt.Sprintf("%#06x\n", d.nvidiaPCIDevice.Class)
```
https://go.dev/play/p/bPr3-R1GtPY

Reply:
Fixed